SPEC.md

```diff
@@ -2972,14 +2991,16 @@ workflow import_structs {
 Example input:
 
 ```json
-{}
+{
+  "import_structs.infile": "hello.txt"
+}
 ```
 
 Example output:
 
 ```json
 {
-  "import_structs.bill": 175000
+  "import_structs.bill": 175000.0
```

Comment:
At least under [the interpretation of JSON that JSON Schema uses](https://json-schema.org/understanding-json-schema/reference/numeric#integer), these are the JSON value.

Are we expecting the runner to produce one serialization here and not the other? Or @stxue1 are you doing something that needs to key on actually having a decimal point?

Reply:
JSON doesn't make a distinction between int and float. The test framework should compare these values as equal.

Reply:
> Are we expecting the runner to produce one serialization here and not the other? Or @stxue1 are you doing something that needs to key on actually having a decimal point?

I currently do try to key on the decimal point to test for floats, but since the JSON schema doesn't actually support a distinction for floats and ints, there isn't much reason to actually key on decimal points, so I'll remove the check from our test suite.